apps/src/code-studio/pd/application_dashboard/summer_workshop_assignment.jsx

```diff
@@ -39,13 +39,12 @@ export default class SummerWorkshopAssignment extends React.Component {
               Assigned Workshop
             </ControlLabel>
           </Col>
-          <Col md={10} sm={9}>
+          <Col md={10} sm={9} style={styles.select}>
```

Comment:
This is just s style fix I noticed along the way. Passing the `maxWidth` from `styles.select` to the `Select` itself rendered funny when it was expanded:

![image](https://user-images.githubusercontent.com/8405316/35713117-81a4f35c-077a-11e8-8716-182702ede9c7.png)